elasticsearch-monitoring/templates/configmap.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ .Values.prometheus.app }}
```

Comment:
Maybe it is better to get a variables labels, where you can give a key/value variable with all the labels you want to  add? You can configure prometheus with other [rulesSelector]https://github.com/coreos/prometheus-operator/blob/master/helm/kube-prometheus/values.yaml#L224

Something like this could work:
```
{{- range $key, $val := .Values.prometheus.labels }}
  {{ $key }}: {{ $val | quote }}
{{- end}}
```

Reply:
Ok, I like that